spec/support/rspec-activemodel-mocks_patch.rb

```diff
@@ -0,0 +1,11 @@
+# A Kaminari patch for scoping the urls.
+require 'rspec/active_model/mocks/version'
+if RSpec::ActiveModel::Mocks::Version::STRING != "1.0.1"
```

Comment:
You can do `Gem.loaded_specs['rspec-activemodel-mocks'].version.to_s`. This seems a bit cleaner because you do not rely on the gems architecture.